src/rfpimp.py

```diff
@@ -427,7 +441,7 @@ def oob_classifier_accuracy(rf, X_train, y_train):
     n_classes = len(np.unique(y))
     predictions = np.zeros((n_samples, n_classes))
     for tree in rf.estimators_:
-        unsampled_indices = _generate_unsampled_indices(tree.random_state, n_samples)
+        unsampled_indices = _get_unsample_indices
```

Comment:
aren't we missing arguments here?  also should be `_get_unsampled_indices` (missing a d)

Reply:
Definitely, I only tested the only occurrence of the function and didn't notice that I had copied without arg. I'll push a new commit with them.

Reply:
cool. Then, I can merge and test with my older version.